modules/gridmap/grid_map_editor_plugin.h

```diff
@@ -176,6 +176,12 @@ class GridMapEditor : public VBoxContainer {
 
 	};
 
+	enum NavigationScheme {
```

Comment:
Can't we reuse `SpatialEditorPlugin::NavigationScheme` instead of duplicating it?

Reply:
Yeah we definitely can and should. My bad